pages/home.jsx

```diff
@@ -1,19 +1,167 @@
-        </Blockquote>
+      <div className="row">
+        <div className="col-sm-offset-2 col-sm-8 col-md-offset-2 col-md-8 col-lg-offset-2 col-lg-8">
+          <div>
+            <Blockquote author="Maurya C. New York, United States"
+                imgSrc="/img/pages/home/maurya-nyc.png" imgSrc2x="/img/pages/home/maurya-nyc@2x.png">
+              <p>"Web literacy is about more than coding - it's about how you can be a better web citizen."</p>
+            </Blockquote>
+          </div>
+        </div>
+      </div>
+    );
+  }
+});
+
+var FeaturedPost = React.createClass({
+  propTypes: {
+    data: React.PropTypes.object.isRequired
+  },
+  render: function() {
+    var parsedDate = moment(this.props.data.publishedDate);
+    return(
+      <div className="featured-post">
+        <div className="entry-posted-container">
+          <p className="entry-posted">
+            <time className="published" dateTime={this.props.data.publishedDate} >
+              <span className="posted-month">{parsedDate.format("MMM")}</span>
+              <span className="posted-date">{parsedDate.format("D")}</span>
+              <span className="posted-year">{parsedDate.format("YYYY")}</span>
+            </time>
+          </p>
+        </div>
+        <div className="entry-header-container">
+          <h3 className="entry-title"><a href={this.props.data.link}>{this.props.data.title}</a></h3>
+          <cite className="author">{this.props.data.author}</cite>
+        </div>
+        <p className="excerpt">
+          {this.props.data.contentSnippet}
+        </p>
+        <a className="more" href={this.props.data.link}>Continue reading</a>
+      </div>
+    );
+  },
+});
+
+var LatestPosts = React.createClass({
+  propTypes: {
+    data: React.PropTypes.array.isRequired
+  },
+  render: function() {
+    return (
+      <ul className="recent-posts">
+        {
+          this.props.data.map(function(post, i) {
+            return (
+              <li key={i}>
+                <a className="post-title" href={post.link}>{post.title}</a>
+                <time className="published" dateTime={post.publishedDate}>
+                  <span>{moment(post.publishedDate).format("MMM D, YYYY")}</span>
+                </time>
+              </li>
+            )
+          })
+        }
+      </ul>
+    );
+  }
+});
+
+var BlogSection = React.createClass({
+  getInitialState: function() {
+    return {
+      featuredPostData: {
+        title: "",
+        author: "",
+        publishedDate: "",
+        contentSnippet: "",
+        link: ""
+      },
+      latestPostsData: []
+    }
+  },
+  loadGoogleAPI: function(callback) {
+    var head = document.getElementsByTagName("head")[0];
+    var gScript = document.createElement("script");
+    gScript.setAttribute("src", "https://www.google.com/jsapi");
+    gScript.onload = callback;
+    head.appendChild(gScript);
+    this.setState({googleAPILoaded: true});
```

Comment:
Er, I just realized that this `googleAPILoaded` state variable isn't actually being used anywhere. Since you're just checking the existence of `window.google` to see if the API is loaded, maybe just get rid of this variable?